src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static checkingHealth() {
+    Animal.alive = Animal.alive.filter(anim => anim.health > 0);
+  }
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = true;
```

Comment:
If we read the task description, we will see that

> Herbivore has a method of hide, which changes the hidden property of the beast to the opposite value and helps to hide from carnivores

So we should change `hidden` to the opposite value (e.g. if it's t`rue` so the method `hide` should change it to the `false`). Now you always change it to `true` only